var/spack/repos/builtin/packages/sqlcipher/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
```

Comment:
```suggestion
```

Reply:
@alalazo before we enforce this, we should make a decision on what to do about `from spack import *`. Should we remove it from existing packages? Should we remove it from the package creation template?

Reply:
Sure, what bothers me most is line 6, the `from` is definitely optional